Update rationale readme

emilybulger opened the pull request:
Update WCAG link to the latest 2.1 version of the spec.

Update header to align with principle.

emilybulger committed:
Update rationale readme

Update WCAG link to the latest 2.1 version of the spec.

Update header to align with principle.

kpeatt commented:
Now that the Mobile app is open, let’s add the removal of the (soon-to-be-) language to this PR and ship it.

emilybulger committed:
Update rationale readme

Align more closely with http://covidshield.app landing page. Remove references to not-yet-released mobile app to reflect open sourcing yesterday. Make clear that COVID Shield is flexible and agnostic enough to be used anywhere, not just Canada.
